Test.C

```diff
@@ -471,20 +529,21 @@ void encryptionStore(struct Player *playerPtr, struct Encryption encryptionStock
     printf("|------------------------------ Encryption Store ------------------------------|\n");
     printf("You currently have %i credits\n", playerPtr->currentCredits);
 
-    printf(" # | Encryption Name      | Cost  | Current Effectiveness\n");
-    printf("---|----------------------|-------|----------------------\n");
+    printf(" # | Encryption Name                        | Cost  | Current Effectiveness\n");
+    printf("---|----------------------------------------|-------|----------------------\n");
     for (int i=0; i<NUM_ENCRYPTION; i++){
-        printf(" %d | %20s | %5d | %.2f\n", i + 1, encryptionStock[i].name, encryptionStock[i].cost, encryptionStock[i].effectiveness);
+        printf(" %d | %40s | %5d | %.2f\n", i + 1, encryptionStock[i].name, encryptionStock[i].cost, encryptionStock[i].effectiveness);
```

Comment:
38 to align with other tables and keep below 80 charcters
```suggestion
        printf(" %d | %38s | %5d | %.2f\n", i + 1, encryptionStock[i].name, encryptionStock[i].cost, encryptionStock[i].effectiveness);
```